.github/workflows/build.yml

```
@@ -94,6 +94,9 @@ jobs:
           x86_64-wrs-vxworks,
           x86_64-unknown-dragonfly,
           x86_64-unknown-haiku,
+          # TODO: once libstd support for cygwin is added
+          # https://github.com/rust-lang/rust/pull/137621
```

```suggestion
          # TODO: once libc support for cygwin is released
          # https://github.com/rust-lang/libc/pull/4308
```
This test only needs libcore support, not libstd support.